jobs/v4/howto/basic_job_sample.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package howto
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	talent "cloud.google.com/go/talent/apiv4beta1"
+	"google.golang.org/api/iterator"
+	talentpb "google.golang.org/genproto/googleapis/cloud/talent/v4beta1"
+)
+
+// [START job_search_create_job]
+
+// createJob create a job as given.
+func createJob(w io.Writer, projectId string, jobToCreate *talentpb.Job) (*talentpb.Job, error) {
```

Comment:
Instead of accepting objects as function parameters, our sample function accept native values (strings, ints, etc) for each of the important fields on the object being created (or updated, etc)

In this case, for [`Job`](https://github.com/googleapis/googleapis/blob/master/google/cloud/talent/v4beta1/job.proto), I would recommend adding a function parameter for every required field:

 - `name` (generated server-side)
 - `company_name` (user-given display name)
 - `requisition_id` (aka. Posting ID)
 - `title`
 - `description`

Because `addresses` is repeated, it could be useful to show how to add multiple addresses with your request. I call this out because the field is marked with:
> Optional but strongly recommended for the best service experience.

In this case, I think it would be OK to accept a String array. (In generated samples we only accept native types, so I would do address1 and address2 to show folks how to construct the request)

FYI: I'm not sure how Go does line breaks in function definitions > 80 characters long
```suggestion
func createJob(w io.Writer, projectId string, companyName string, requisitionId string,
	title string, description string, address1 string, address2 string)
	(*talentpb.Job, error) {
```

Reply:
Would this be a repetition of what's in [custom_attributes_sample.go](jobs/v4/howto/custom_attribute_sample.go)?